awx/ui/client/src/standard-out/inventory-sync/standard-out-inventory-sync.route.js

```diff
@@ -20,7 +20,8 @@ export default {
     data: {
         socket: {
             "groups":{
-                "jobs": ["status_changed"]
+                "jobs": ["status_changed"],
```

Comment:
When it's all said and done, we'll want the "summary" in the `jobs` array as well, but this is fine for now.